src/sync/branches.ts

```diff
@@ -3,20 +3,22 @@ import { GitHubAPI } from "probot";
 import { Repository } from "models/subscription";
 import { GitHubInstallationClient } from "../github/client/github-installation-client";
 import { LoggerWithTarget } from "probot/lib/wrap-logger";
+import { numberFlag, NumberFlags } from "config/feature-flags";
 
 // TODO: better typings
 export const getBranchTask = async (
 	logger: LoggerWithTarget,
 	_github: GitHubAPI,
 	newGithub: GitHubInstallationClient,
-	_jiraHost: string,
+	jiraHost: string,
 	repository: Repository,
 	cursor?: string | number,
 	perPage?: number) => {
 	// TODO: fix typings for graphql
 	logger.info("Syncing branches: started");
 	perPage = perPage || 20;
-	const result = await newGithub.getBranchesPage(repository.owner.login, repository.name, perPage, cursor as string);
+	const timeCutoff = await numberFlag(NumberFlags.SYNC_BRANCH_COMMIT_TIME_LIMIT, NaN, jiraHost);
```

Comment:
Curious why the default value is NaN, not `undefined` ?

Reply:
Mentioned below.